aries-site/src/components/home/Featured.js

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from 'react';
+
+import { Box, Grid, Paragraph, ResponsiveContext, Stack, Text } from 'grommet';
+import Link from 'next/link';
+import { ContentPreviewCard } from '../cards';
+import { nameToPath } from '../../utils';
+import { featured } from '../../data';
+
+const FeaturedLayout = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
+      fill
+      pad={{
+        horizontal: size !== 'small' ? 'xlarge' : 'large',
+        bottom: 'small',
+      }}
+    >
+      <Grid
+        rows={[['auto', 'full']]}
+        columns={{ count: 'fit', size: '300px' }}
```

Comment:
```suggestion
        columns={{ count: 'fit', size: '300px' }}
```
any way we can rely on t-shirt sizing here instead? 'medium'

Reply:
That's what I originally had, but it makes the cards a bit to wide for this spot (Vicky commented on it earlier.) Thoughts?

Reply:
I pulled it up locally to compare, I think it just affects where the breakpoint is with Grid -- so if the window is slightly condensed there is a window width that allows the Cards to be a bit wider than the design may call for.

I think in this case, I'd push for using t-shirt size "medium", since using t-shirt sizing is something we're advocated other teams do also. @vavalos5 are you okay with that?

Reply:
Yes! "Medium" makes sense

Reply:
Done